lib/vanagon/platform/windows.rb

```diff
@@ -289,6 +289,105 @@ def output_dir(target_repo = "")
         File.join("windows", target_repo, @architecture)
       end
 
+      # Generate correctly formatted wix elements that match the
+      # structure of the directory input
+      #
+      # @param services, Array of components services
+      # and optionally:
+      # @return [string] correctly formatted wix element string
+      def generate_wix_dirs(services)
+        directories = []
+        services.map { |svc| directories.push({ :path => svc.service_file, :id => "#{svc.id}BINDIR" }) }
```

Comment:
So, well get `:path => "SourceDir\\#{settings[:base_dir]}\\#{settings[:company_id]}\\#{settings[:product_id]}\\puppet\\bin\\nssm.exe"` and `:id => PXPAGENTBINDER`?

If we pass it a file, will it just take the parent directory and assign the ID to that? How does it know which ID to put? It looks like were going to have `PXPAGENTBINDIR`, `PUPPETBINDIR`, and `MARIONETTECOLLECTIVEBINDIR` all correspond to the same directory, if I'm still following everything?